core/src/main/resources/reference.conf

```diff
@@ -392,7 +392,25 @@ services {
   MetadataService {
     class = "cromwell.services.metadata.impl.MetadataServiceActor"
     # Set this value to "Inf" to turn off metadata summary refresh.  The default value is currently "2 seconds".
+    # FIXME: move
     # metadata-summary-refresh-interval = "Inf"
+    #
+    # config {
+      # For higher scale environments, e.g. many workflows and/or jobs, DB write performance for metadata events
+      # can improved by writing to the database in batches. Increasing this value can dramatically improve overall
+      # performance but will both lead to a higher memory usage as well as increase the risk that metadata events
+      # might not have been persisted in the event of a Cromwell crash.
+      #
+      # For normal usage the default value of 1 (effectively no batching) should be fine but for larger/production
+      # environments we recommend a value of at least 500. There'll be no one size fits all number here so testing
```

Comment:
What about benchmarking? or actually "benchmarking & tuning" or something like that, i.e. making it clear that one should have concrete data to inform the tuning